draft-ietf-core-target-attr.md

```diff
@@ -98,9 +96,9 @@ The current revision of that specification clarifies ({{Section 2.2 of -linking}
    registries of target attributes.
 
 This document introduces an IANA registry for coordinating names of target
-attributes when used in Constrained RESTful Environments, with
-specific instructions for the designated expert for this registry ({{de-instructions}}).
-It updates the RD Parameters Registry of {{-rd}} to coordinate with
+attributes when used in CoRE, with
+specific instructions for the Designated Expert for this registry ({{de-instructions}}).
+It adds a note to the RD Parameters Registry of {{-rd}} to coordinate with
```

Comment:
It's not just a note, it is a normative change to how the 9176 registry operates.

Reply:
This does not change how 9176 registry operates (registry new entries), this only adds a note for the IANA to trigger an action on another registry.

I can cite many examples, but you can refer, for example, to https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml where there is a note to trigger an IANA action on another registry:

>    Note: When this registry is modified, the YANG module [[iana-dns-class-rr-type](https://www.iana.org/assignments/iana-dns-class-rr-type)] 
>     must be updated as defined in [[RFC9108](https://www.iana.org/go/rfc9108)].

That is similar to what you have in the draft. It is simply that the nature of the IANA triggered action that varies.